stable/redis/values.yaml

```diff
@@ -0,0 +1,33 @@
+## Bitnami Redis image version
+## ref: https://hub.docker.com/r/bitnami/redis/tags/
+##
+image: bitnami/redis:3.2.4-r0
```

Comment:
Yah curious on this one as well. I know that part of it is that our friends at Bitnami are helping us wit some charts